src/index.css

```
@@ -139,43 +139,37 @@ html {
+	white-space: normal !important;
+	overflow-wrap: break-word;
 }
 
+div[class*='react-hot-toast'] > div,
 div[class*='react-hot-toast'] > div > div {
-	white-space: nowrap !important;
-	min-width: fit-content !important;
-	width: auto !important;
+	min-width: 0;
+	white-space: normal !important;
+	word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)